src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -110,12 +110,15 @@ $pf-c-tree-view--MaxNesting: 10;
 }
 
 .pf-c-tree-view__node-toggle {
+  position: absolute;
+  left: calc(var(--pf-c-tree-view__node--PaddingLeft) - var(--pf-global--spacer--lg) - var(--pf-global--spacer--lg));
```

Comment:
need a var for this

Reply:
Couldn't this just be `--pf-c-tree-view__node--PaddingLeft`, then use `transform: translateX(-100%);`?